drools-ha/ha-core-infra/src/test/java/org/kie/hacep/PodAsReplicaTest.java

```diff
@@ -34,7 +35,7 @@
 
     private Logger logger = LoggerFactory.getLogger(PodAsReplicaTest.class);
 
-    @Test(timeout = 30000L)
+    @Test(timeout = 600000L) //see random failure https://github.com/kiegroup/droolsjbpm-integration/pull/2156
```

Comment:
We probably don't need the link to the PR where it failed anymore; if it fails again despite the increased timeout, we will notice.